### PR TITLE
feat: sqlite aggregator

### PR DESCRIPTION
**Pull request by ToniRamirezM**
## Description

Use sqlite instead of postgresql in the aggregator.

### CODE REVIEW

File: test/config/test.config.toml [hunk 1]
```diff
@@ -58,14 +58,7 @@ AggLayerURL = ""
...
 SequencerPrivateKey = {}
-	[Aggregator.DB]
-		Name = "aggregator_db"
-		User = "aggregator_user"
-		Password = "aggregator_password"
-		Host = "cdk-aggregator-db"
-		Port = "5432"
-		EnableLog = false	
-		MaxConns = 200
+DBPath = ""
```
Comment (joanestebanr) [inline test/config/test.config.toml:61]: Setting to "" is not going to fail the execution?

**Review by joanestebanr — APPROVED** [review]

**Review by goran-ethernal — APPROVED** [review]

File: aggregator/db/migrations/0001.sql [hunk 2]
```diff
@@ -1,32 +1,22 @@
 -- +migrate Down
```
Comment (Stefan-Ethernal) [inline aggregator/db/migrations/0001.sql:1]: Should we drop `proof` and `sequence` tables here?

File: aggregator/db/dbstorage/proof.go [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+	_, err := e.Exec(
+		addGeneratedProofSQL, proof.BatchNumber, proof.BatchNumberFinal, proof.Proof, proof.ProofID,
+		proof.InputProver, proof.Prover, proof.ProverID, generatingSince, createdAt, updatedAt,
+	)
+	return err
+}
+
+// UpdateGeneratedProof updates a generated proof in the storage
+func (d *DBStorage) UpdateGeneratedProof(ctx context.Context, proof *state.Proof, dbTx db.Txer) error {
+	const addGeneratedProofSQL = `
```
Comment (Stefan-Ethernal) [inline aggregator/db/dbstorage/proof.go:260]:
```suggestion
	const updateGeneratedProofSQL = `
```

File: aggregator/db/migrations/0001.sql [hunk 4]
```diff
@@ -1,32 +1,22 @@
...
-DROP SCHEMA IF EXISTS aggregator CASCADE;
 
 -- +migrate Up
```
Comment (Stefan-Ethernal) [inline aggregator/db/migrations/0001.sql:3]: Do we need a schema (e.g. named `aggregator`) as well, wdyt? Although the aggregator db is a separate file, so maybe it doesn't make sense at this point...

File: aggregator/db/migrations/0001.sql [hunk 5]
```diff
@@ -1,32 +1,22 @@
...
-);
-
-CREATE TABLE IF NOT EXISTS aggregator.proof (
-	batch_num BIGINT NOT NULL REFERENCES aggregator.batch (batch_num) ON DELETE CASCADE,
 	batch_num_final BIGINT NOT NULL,
 	proof varchar NULL,
 	proof_id varchar NULL,
 	input_prover varchar NULL,
 	prover varchar NULL,
 	prover_id varchar NULL,
```
Comment (Stefan-Ethernal) [inline aggregator/db/migrations/0001.sql:11]: Nitpick: it is probably more in a fashion of SQL Lite (https://www.sqlite.org/datatype3.html)
```suggestion
	proof TEXT NULL,
	proof_id TEXT NULL,
	input_prover TEXT NULL,
	prover TEXT NULL,
	prover_id TEXT NULL,
```

**Review by Stefan-Ethernal — APPROVED** [review]
LGTM, leaving some optional comments to consider.

File: aggregator/config.go [hunk 6]
```diff
@@ -117,8 +108,8 @@ type Config struct {
 	// UseFullWitness is a flag to enable the use of full witness in the aggregator
 	UseFullWitness bool `mapstructure:"UseFullWitness"`
 
-	// DB is the database configuration
-	DB db.Config `mapstructure:"DB"`
+	// DBPath is the path to the database
+	DBPath string `mapstructure:"DBPath"`
```
[the remark this answers is not in the document]
Comment (ToniRamirezM) [reply aggregator/config.go:112]: Well, I used DBPath just to keep the same name than on other components.

**Pull request merged**